ios/BT/ExposureManager.swift

```diff
@@ -536,10 +440,103 @@ private extension ExposureManager {
     }
   }
 
-  func cleanup() {
-    // Delete downloaded files from file system
-    localUncompressedURLs.cleanup()
-    localUncompressedURLs = []
-    downloadedPackages = []
+  // MARK: == Exposure Detection Private Promisses ==
+
+  func fetchFilesIndexes() -> Promise<String> {
+    return Promise<String> { fullfill, reject in
+      self.apiClient.requestString(IndexFileRequest.get,
+                              requestType: .downloadKeys) { result in
+        switch result {
+        case .success(let index):
```

Comment:
`index` might be an odd name for this. What about something such as `keyArchiveFilePathsString`? That is also quite a long variable name, but I'd like to communicate the meaning of the variable here somehow 🤔

Reply:
you are right, I'll change it